google/cloud/storage/benchmarks/storage_file_transfer_benchmark.cc

```diff
@@ -0,0 +1,242 @@
+  auto meta =
+      client
+          .CreateBucket(bucket_name,
+                        gcs::BucketMetadata()
+                            .set_storage_class(gcs::storage_class::Regional())
+                            .set_location(options.region),
+                        gcs::PredefinedAcl("private"),
+                        gcs::PredefinedDefaultObjectAcl("projectPrivate"),
+                        gcs::Projection("full"))
+          .value();
+  std::cout << "# Running test on bucket: " << meta.name() << "\n";
+  std::string notes = google::cloud::storage::version_string() + ";" +
+                      google::cloud::internal::compiler() + ";" +
+                      google::cloud::internal::compiler_flags();
+  std::transform(notes.begin(), notes.end(), notes.begin(),
+                 [](char c) { return c == '\n' ? ';' : c; });
+  std::cout << "# Start time: "
+            << gcs::internal::FormatRfc3339(std::chrono::system_clock::now())
+            << "\n# Region: " << options.region
+            << "\n# Duration: " << options.duration.count() << "s"
+            << "\n# File Size: " << options.file_size
+            << "\n# File Size (MiB): " << options.file_size / gcs_bm::kMiB
+            << "\n# Download buffer size (KiB): "
+            << options.download_buffer_size / gcs_bm::kKiB
+            << "\n# Upload buffer size (KiB): "
+            << options.upload_buffer_size / gcs_bm::kKiB
+            << "\n# Build info: " << notes << "\n";
+
+  std::cout << "# Creating file to upload ..." << std::flush;
+  auto filename = gcs_bm::MakeRandomFileName(generator);
+  std::ofstream(filename, std::ios::binary)
+      << gcs_bm::MakeRandomData(generator, options.file_size);
+  std::cout << " DONE\n"
+            << "# File: " << filename << "\n"
+            << std::flush;
+
+  auto deadline = std::chrono::system_clock::now() + options.duration;
+  for (auto now = std::chrono::system_clock::now(); now < deadline;
+       now = std::chrono::system_clock::now()) {
+    auto object_name = gcs_bm::MakeRandomObjectName(generator);
+
+    auto upload_start = std::chrono::steady_clock::now();
+    auto object_metadata =
+        client.UploadFile(filename, bucket_name, object_name);
+    auto upload_elapsed = std::chrono::duration_cast<std::chrono::nanoseconds>(
+        std::chrono::steady_clock::now() - upload_start);
+    double gbps = options.file_size * 8.0 / upload_elapsed.count();
+    auto ms =
+        std::chrono::duration_cast<std::chrono::milliseconds>(upload_elapsed);
+    auto MiBs =
+        (double(options.file_size) / gcs_bm::kMiB) / (ms.count() / 1000.0);
+    std::cout << "FileUpload," << options.file_size << ','
+              << upload_elapsed.count() << ',' << gbps << ',' << ms.count()
+              << ',' << MiBs << ',' << object_metadata.status().code() << "\n";
+    if (!object_metadata) {
+      std::cout << "# Error in FileUpload: " << object_metadata.status()
+                << std::endl;
```

Comment:
s/std::endl/"\n"/

Reply:
Done.

Reply:
I still see it, I think.

Reply:
Doh! I removed another one, fixed for realsies now.